src/Hero/Hero.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+
+/**
+ * Hero Component
+ *
+ * HTML card element
+ */
+
+const HeroRoot = ({
+  bgImage: { src } = {},
```

Comment:
If you are not using any other values than src why make this parameter an object?